walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -24,13 +70,15 @@ img {
 
 .header-container {
   align-items: center;
+  background-color: grey;
   display: flex;
   height: 100px;
   justify-content: space-evenly;
+  margin-bottom: 10px;
   margin-right: auto;
   margin-left: auto;
   max-width: 1000px;
-  width: 100%;
+  width: 100%;  
```

Comment:
remove trailing whitespace (hint: use an auto formatter)

Reply:
formatter took care of this